learn-to-use-vim/10-introduction-to-vim-plugins.md

```diff
@@ -0,0 +1,12 @@
+# Introduction to Vim Plugins
```

Comment:
I think it would be really nice to have an example of how to use Plugins Managers or just how to add a plugin using a manager would be very beneficial.